_rouses/Creed Ford.html

```diff
@@ -9,3 +9,6 @@
 <a href="https://www.youtube.com/channel/UCwXosaHl2P-HaVWNxhJTW7Q"> My super awesome skiing channel!</a>
 <img src="/assets/images/Apple_4.0.png" class="with-caption img-fluid">
 <p>This is something I drew on my ipad.</p>
+<p>My favorite food is Sushi!</p>
+<p>My birthday is March 22, 2007.</p>
+<p>Here is a picture of one of my favorite ski runs at sundance</p>
```

Comment:
If you want a photo to appear here, you'll need to add an `<img>` tag like you did for the other photos. I'm happy to pair with you if you need some help!